### PR TITLE
Change example folder

### PR DESCRIPTION
**Pull request by rodolfo-viana**
Just changed /tmp/serenata-data for /data/, which is the folder we get when we install Serenata de Amor.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -37,19 +37,19 @@ Copy `config.ini.example` as `config.ini` and edit it with your own credentials.
...
-  datasets = Datasets('/tmp/serenata-data/')
+  datasets = Datasets('data/')
 
   # now lets see what are the latest datasets available
   for dataset in datasets.downloader.LATEST:
       print(dataset)  # and you'll see a long list of datasets!
 
   # and let's download one of them
-  datasets.downloader.download('2016-12-06-reibursements.xz')  # yay, you've just downloaded this dataset to /tmp/serenata-data/
+  datasets.downloader.download('2016-12-06-reibursements.xz')  # yay, you've just downloaded this dataset to /data/
```
Comment (cuducos) [inline README.rst]: Actually `/data/` would be the `data` directory in the _root_ of your volume. The correct way would be `data/` (i.e. the data directory inside the current working directory).

Comment (rodolfo-viana) [inline README.rst]: Should I correct that? I never know what I am supposed to do next. :)

Comment (cuducos) [inline README.rst]: Yes, please ; )

Just commit it with a new message and push to your repo/branch that the change reflects here.

Comment (rodolfo-viana) [inline README.rst]: Done.

File: README.rst [hunk 2]
```diff
@@ -37,19 +37,19 @@ Copy `config.ini.example` as `config.ini` and edit it with your own credentials.
 Example 1: How do I download the datasets?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We have `plenty of them <https://github.com/datasciencebr/serenata-de-amor/blob/master/CONTRIBUTING.md#datasets-data>`_ ready for you to download from our servers. And this toolbox helps you get them. Let's say you want your datasets at `/tmp/serenata-data/`:
+We have `plenty of them <https://github.com/datasciencebr/serenata-de-amor/blob/master/CONTRIBUTING.md#datasets-data>`_ ready for you to download from our servers. And this toolbox helps you get them. Let's say you want your datasets at `/data/`:
```
Comment (cuducos) [inline README.rst]: Here we still have `/data/` instead of `data/`.

**Comment by jtemporal** [comment]
closing this in favor of #109

**Pull request closed without merging**